lib/tasks/translations.rake

```diff
@@ -184,4 +184,44 @@ namespace :translations do
       File.rename translation_file, OpenProject::Translations::Engine.root.join('config', 'locales', "#{js_translation?(translation_file) ? 'js-' : ''}#{language_key}.yml")
     end
   end
+
+  desc "Update Gemfile.lock to newest Version of OpenProject Translation"
+  task :update_gemfile_lock => :environment do
+    target_path = 'tmp/openproject'
+    branch = ENV['Translation_working_branch']
```

Comment:
This is a remnant from [old times](https://github.com/finnlabs/ci/blob/dev/ci.sh#L316)